packages/quantic/force-app/main/default/lwc/quanticColoredResultBadge/quanticColoredResultBadge.js

```diff
@@ -20,6 +19,23 @@ export default class QuanticColoredResultBadge extends LightningElement {
    * @type {string}
    */
   @api label;
+  /**
+   * The [result item](https://docs.coveo.com/en/headless/latest/reference/search/controllers/result-list/#result).
+   * @api
+   * @type {Result}
+   */
+  @api result;
+  /**
+   * The [result field](https://docs.coveo.com/en/headless/latest/reference/search/controllers/result-list/#result).
```

Comment:
Is it on purpose that the link goes to the same place?
If I read the code correctly, you are only looking in the ".raw" so this would be more appropriate?
https://docs.coveo.com/en/headless/latest/reference/search/controllers/result-list/#raw

Reply:
I've followed the same thing than here: [quanticResultField](https://github.com/coveo/ui-kit/blob/master/packages/quantic/force-app/main/default/lwc/quanticResultField/quanticResultField.js)